tests/rpc/test_fiat_convert.py

```diff
@@ -21,6 +22,12 @@ def test_fiat_convert_is_supported(mocker):
 def test_fiat_convert_find_price(mocker):
     fiat_convert = CryptoToFiatConverter()
 
+    fiat_convert._cryptomap = {}
+    fiat_convert._backoff = 0
+    mocker.patch('freqtrade.rpc.fiat_convert.CryptoToFiatConverter._load_cryptomap',
+                 return_value=None)
+    assert fiat_convert.get_price(crypto_symbol='BTC', fiat_symbol='EUR') == 0.0
```

Comment:
I am not really sure whether this belongs here or not. We test find_price, yes, but we also patch something out that other tests below might need to produce meaningful tests. But we also patch out the tested method there and test the hardcoded return value, so it might be okay 😸

Reply:
Mocks are "per function / per test" - so the mock will return to norma for the next test.

I think it's perfectly fine - and adds a realistic case (where it's returning empty).